test/unit/tasks/test_calculate_kpi.py

```diff
@@ -143,6 +142,16 @@ def test_calc_kpi_no_prediction_exception(self):
                 prediction_job["id"], realised_load, pd.DataFrame(), realised_load
             )
 
+    def test_calc_kpi_empty_basecase(self):
+        """An empty basecase should not return an exception
+        Instead, basecase metrics should be NaN"""
+
+        kpis = calc_kpi_for_specific_pid(
+            prediction_job["id"], realised_load, predicted_load, basecase=pd.DataFrame()
+        )
+        # Assert KPIs
+        kpis  # TODO improve
```

Comment:
Maybe for now just check if the result is a dict
```suggestion
        isinstance(kpis  , dict)# TODO improve
```